OAuthv2/Microsoft Identity Platform OAuth2.postman_collection.json

```diff
@@ -0,0 +1,1453 @@
+									"value": "code"
+								},
+								{
+									"key": "redirect_uri",
+									"value": "http://localhost/myapp/"
+								},
+								{
+									"key": "response_mode",
+									"value": "query"
+								},
+								{
+									"key": "scope",
+									"value": "openid%20offline_access%20https%3A%2F%2Fgraph.microsoft.com%2Fuser.read"
+								},
+								{
+									"key": "state",
+									"value": "12345"
+								}
+							]
+						},
+						"description": "A sign in request to begin the OAuth 2.0 code flow.  Be sure to copy & paste into a browser! Running this request in Postman will just return you the HTML of our login pages."
+					},
+					"response": []
+				},
+				{
+					"name": "Token Request - Auth Code",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/x-www-url-form-urlencoded"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "client_id",
+									"value": "{{ClientID}}",
+									"type": "text"
+								},
+								{
+									"key": "scope",
+									"value": "https://graph.microsoft.com/user.read",
+									"type": "text"
+								},
+								{
+									"key": "redirect_uri",
+									"value": "http://localhost/myapp/",
+									"type": "text"
+								},
+								{
+									"key": "grant_type",
+									"value": "authorization_code",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "{{ClientSecret}}",
+									"type": "text"
+								},
+								{
+									"key": "code",
+									"value": "{{replace-with-code-copied-from-browser}}",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
+							"protocol": "https",
+							"host": [
+								"login",
+								"microsoftonline",
+								"com"
+							],
+							"path": [
+								"common",
+								"oauth2",
+								"v2.0",
+								"token"
+							]
+						},
+						"description": "The POST request to exchange an auth code for an access token.  Be sure to replace the code with your own, that you recieved after signing in!"
+					},
+					"response": []
+				},
+				{
+					"name": "Microsoft Graph Request",
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{the-access-token-copied-from-postman}}"
+							}
+						],
+						"url": {
+							"raw": "https://graph.microsoft.com/v1.0/me",
+							"protocol": "https",
+							"host": [
+								"graph",
+								"microsoft",
+								"com"
+							],
+							"path": [
+								"v1.0",
+								"me"
+							]
+						},
+						"description": "An example use of the access_token.  Note: it will only work for users with a valid mailbox.  Be sure to replace the access token with the one you got from the token request!"
```

Comment:
```suggestion
						"description": "An example use of the access_token.  It will work only for users with a valid mailbox. Replace the access token with the one you got from the token request."
```